project-p1-first-webpage-master/code/index.html

```diff
@@ -0,0 +1,108 @@
+  </div>
+  <div class="column" style="background-color:rgba(119, 161, 203, 0.5);">
+        <h3>Elterncommunity in Zürich</h3>
+         <p>Facebook ist dir zu öffentlich, WhatsApp zu Beginn zu privat? Uns auch! Deswegen brauchen wir ein eigenes lokales Netzwerk, wo wir uns austauschen können. Lass dich inspirieren und stärken und teile deine eigenen Fragen und Erfahrungen rund ums ‚Elternsein"</p>
+  </div><div class="column" style="background-color:rgb(255, 172, 129, 0.5);">
+       <h3>Eltern in deiner Nähe</h3>
+         <p>Eine digitale Community etwas Tolles, insbesondere, wenn du dich mit anderen Eltern in deiner Nähe vernetzen kannst um gemeinsam etwas zu unternehmen - gemeinsam den Spielplatz unsicher machen oder aber einfach zusammen einen kinderlosen Abend geniessen.</p>
+  </div>   
+</div>
+
+
+<div class="A5">
+  <img src="kaffe.jpg" alt="Kaffe trinken" style="float:right;width:350px;height:200px">
+  <h2 id="T5">Meetup</h2>
+    <p class="textmeetup">  Wir wollen zusammenstehen und uns gegenseitig unterstützen - nicht nur digital, sondern auch analog. Einmal im Monat treffen wir uns dazu in entspannter Atmosphäre in Zürich zum Kaffee oder Apèro.
+Unsere Gruppe ist offen für Mütter/Väter/pflegende Angehörige von behinderten Kindern.
+Hier findest du die nächsten Termine:
+<a href="https://www.meetup.com/de-DE/4parents-Eltern-von-Kindern-mit-Behinderung-Gruppe-Zurich/" target="_blank"> Meetup</a>
+    </p>
+</div>
+
+
+<div class="A6">
+  <h2 id="T6">About us - unsere Geschichte </h2>
+
+    <div class="right"></div>
+      <img src="kp.jpg" style="float:left;width: auto;height:auto;padding: 30px;">
+    </div>
+    <div class="left">
+     <p>Wir, das sind Kanza, Patricia (und Bettina). Seit der Geburt unserer Kinder hat sich unser Leben um 180 Grad gedreht: wir haben unglaubliche Kräfte mobilisiert, 
+      haben gelernt, mit Ungewissheit umzugehen, das Unvorstellbare auszuhalten, wie etwa nicht zu wissen, wie es unseren Kindern wirklich geht. Ein klarer Fall von Superpowers, oder? 
+      Wir hatten aber auch plötzlich neue Ängste und Fragen, die uns nur andere betroffene Eltern beantworten konnten und wir sehnten uns nach sozialen Kontakten und neuen Freundschaften mit Mamas und Papas in der Nähe für Austausch, gemeinsame Treffen und gegenseitige Unterstützung. Uns begegneten Mütter oder Väter – (z.B.!) alleine Rollstuhl-schiebend – und wir wünschten uns, den Mut zu haben, sie anzusprechen und kennen zu lernen. 
+      Selber haben wir uns über Umwege online gefunden und kurz darauf zum ersten Mal persönlich getroffen. 
+      Gerne würden wir diese Erfahrung mit anderen Eltern teilen. Aus diesem Grund haben wir 4parents gegründet.
+      </p>
+    </div>
+</div>
+
+  </body>
+  
+    <footer class="footer"> Get in touch! 
+    </footer>
```

Comment:
Oops! All of your HTML content should be included in the `body` so the closing tag should be after the `</footer>` tag on line 92